app/models/orchestration_template.rb

```diff
@@ -153,7 +153,7 @@ def md5=(_md5)
   end
 
   def self.calc_md5(text)
-    Digest::MD5.hexdigest(text) if text
+    Rails.application.config.digest_class.hexdigest(text) if text
```

Comment:
We need a migration every time when we switch to another digest_class. It is not doable if we allow the user to configure the class.